src/ensembl/src/content/app/browser/drawer/drawer-views/transcript-summary/TranscriptSummary.scss

```diff
@@ -0,0 +1,82 @@
+@import 'src/styles/common';
```

Comment:
Comaring current implementation:

![image](https://user-images.githubusercontent.com/6834224/107391581-bb2e4880-6af0-11eb-9228-408be175e8e1.png)

With the design in the XD:
![image](https://user-images.githubusercontent.com/6834224/107391637-cb462800-6af0-11eb-97de-42cb4daf1949.png)

- first line in the XD (Transcript) seems to be in its own group. Don't know why the distance between that line and the next group of lines (transcript name, transcript length, exons) is different than the distance between this second group and the third group. Maybe confirm w/Andrea
- missing bp after transcript length
- transcript length not formatted
- I have a suspicion that the light-grey colour in the XD may be achieved through the use of a thinner font rather than of the same font with a different colour. Confirm w/Andrea

Reply:
All, except first and the last are done. I will check the spacing with Andrea during the code review.

Reply:
That's all done now.